### DISCUSSION TITLE
Can't use HTML tag in Settings tab

### DISCUSSION THREAD

**Discussion by MarcoAppleby** [globaleaks/globaleaks-whistleblowing-software#3473] [Q&A] [answered]
I want to insert an ancor tag to link the privacy policy before the user can proceed to compile the form.

But it is impossible to use html tag in the setting forms.

I have the last version of globaleaks: 4.11.15.

I rememebr that in a previous version it was possibile.

**Comment by gianlucagilardi** [comment, marked as answer]
Have you tried using Markdown instead of HTML to add a link?

**Comment by MarcoAppleby** [comment]
I've tried now. With Markdown it works.

Thanks
